tsc/meetings/2019-07-11.md

```diff
@@ -0,0 +1,122 @@
+Minutes from 23rd OpenVDB TSC meeting, July 11, 2019
+
+Attendees: *Dan* B., *Peter* C., *Ken* M.
+
+Absent: *Nick* A., *Jeff* L.
+
+Additional Attendees: Bruce Chernia (Intel), John Mertic (LF)
+
+Agenda:
+
+1) Quorum
+2) Secretary
+3) Round-Robin: Current and upcoming PRs
+4) Graduation
+5) Next Meeting
+
+
+1) Quorum was confirmed.
+
+2) Secretary was Dan Bailey
+
+3) Round-Robin: Current and upcoming PRs
+
+Ken:
+
+Fast sweeping, value extension tool to be shared very soon. Initially this will
+just be the core library implementation with subsequent discussion about the
+right approach for a Houdini integration to follow.
+
+Peter:
+
+PR 455 - Some issues raised on the Sharpening PR that need consideration.
+Optimization to repopulate stencil in Z-axis could be generalized to more axes
+and extended to support jumps of more than one voxel. Also, Peter has been asked
+by artists at DWA for a more generic convolution kernel and this PR could be
+seen as a proof of concept of doing that.
+
+Ken proposed (and has already implemented) an alternative technique which is
+well established (actually dates back to the 1930s) and often referred to as
+"unsharp masking". Peter wasn't able to get satisfactory results using this
+technique, however that does not mean that it doesn't work - it certainly does
+for Ken. However, it suggests that Peter might have found a better solution
+which of course remains to be proven. Both methods are based on separable filter
+kernels so Ken doesn't see why there should be a significant difference in the
+computational efficiency, but cannot speak to the difference in quality since he
+hasn't had a change to do a direct comparison.
+
+The current implementation only works on level sets, but could be relatively
+easily extended to support fog volumes and vector fields. Ken mentioned that
+vector fields would be very useful to him. One option is to complete the current
+PR and then introduce a new PR for the extension.
+
+Peter didn't have access to his list of forthcoming work.
+
+Dan:
+
+PR 487 - Fixes CI issue where Houdini builds are skipped rather than fail when
+run outside of ASWF Azure account. Trivial fix, anyone can approve.
+
+PR 483 - Adds code coverage support to CMake, Nick currently reviewing. Minor
+issue where code coverage takes a very long time to run (2 hours+) so can only
+be run sporadically. A number of options to resolve this. Ken suggested running
+the unit tests concurrently, though there may be issues if the tests themselves
+run in parallel. Dan suggested using a smaller data set for some of the heavy
+tests, perhaps with a flag that enables the more expensive tests for a less
+frequent test run. Peter suggested using a mode that he added to the unit tests
+where you can supply a file that defines which unit tests to execute would allow
+for filtering out some of the expensive ones.
```

Comment:
My suggestion was actually to use the file mechanism to partition the tests into groups to be run concurrently (i.e., across multiple `vdb_test` processes).  I'm not in favor of excluding tests as a standard procedure.

Reply:
Just addressed this, thanks and sorry for the delay.